--- maxima-5.9.3.99rc1/src/Makefile.am.sbcl-disable-debugger	2006-07-31 21:40:16.000000000 -0500
+++ maxima-5.9.3.99rc1/src/Makefile.am	2006-08-01 09:09:51.000000000 -0500
@@ -151,7 +151,7 @@
 
 
 if SBCL
-EXECUTESBCL = $(SBCL_NAME) --noinform --noprint --eval
+EXECUTESBCL = $(SBCL_NAME) --noinform --noprint --disable-debugger --eval
 all-local: binary-sbcl/maxima.core
 install-exec-local: install-sbcl
 uninstall: uninstall-sbcl
--- maxima-5.9.3.99rc1/src/Makefile.in.sbcl-disable-debugger	2006-07-31 21:46:46.000000000 -0500
+++ maxima-5.9.3.99rc1/src/Makefile.in	2006-08-01 09:10:46.000000000 -0500
@@ -201,7 +201,7 @@
 @CMUCL_TRUE@EXECUTECMUCL = $(CMUCL_NAME) -noinit -batch
 @SCL_TRUE@EXECUTESCL = $(SCL_NAME) -noinit -batch
 @ACL_TRUE@EXECUTEACL = $(ACL_NAME) -batch
-@SBCL_TRUE@EXECUTESBCL = $(SBCL_NAME) --noinform --noprint --eval
+@SBCL_TRUE@EXECUTESBCL = $(SBCL_NAME) --noinform --noprint --disable-debugger --eval
 @GCL_TRUE@EXECUTEGCL = $(GCL_NAME) -batch -eval
 @OPENMCL_TRUE@EXECUTEOPENMCL = $(OPENMCL_NAME) -e
 genericdir = $(instsrcdir)
