diff -up maxima-5.28.0/src/maxima.in.clisp-noreadline maxima-5.28.0/src/maxima.in
--- maxima-5.28.0/src/maxima.in.clisp-noreadline	2012-06-15 00:21:34.000000000 -0500
+++ maxima-5.28.0/src/maxima.in	2012-08-19 14:48:43.879825772 -0500
@@ -103,6 +103,7 @@ while [ -n "$1" ]; do
 	--userdir ) : ; shift;; # noop; handled by process_userdir_argument
 	--userdir=*) : ;; # noop; handled by process_userdir_argument
 	-v|--verbose ) verbose=true;;
+	--disable-readline ) CLISP_OPTIONS="-disable-readline";;
 	*) ;;
     esac
     shift
@@ -136,9 +137,9 @@ fi
 if [ "$MAXIMA_LISP" = "clisp" ]; then
     if [ "$layout_autotools" = "true" ]; then
       if [ -x "$MAXIMA_IMAGESDIR/binary-$MAXIMA_LISP/maxima" ]; then
-        exec "$MAXIMA_IMAGESDIR/binary-$MAXIMA_LISP/maxima" $MAXIMA_LISP_OPTIONS -q "" -- "$arg1" "$arg2" "$arg3" "$arg4" "$arg5" "$arg6" "$arg7" "$arg8" "$arg9"
+        exec "$MAXIMA_IMAGESDIR/binary-$MAXIMA_LISP/maxima" $MAXIMA_LISP_OPTIONS $CLISP_OPTIONS -q "" -- "$arg1" "$arg2" "$arg3" "$arg4" "$arg5" "$arg6" "$arg7" "$arg8" "$arg9"
       else
-	exec "$MAXIMA_IMAGESDIR/binary-$MAXIMA_LISP/@CLISP_RUNTIME@" $MAXIMA_LISP_OPTIONS -q -M "$maxima_image_base.mem" "" -- "$arg1" "$arg2" "$arg3" "$arg4" "$arg5" "$arg6" "$arg7" "$arg8" "$arg9"
+	exec "$MAXIMA_IMAGESDIR/binary-$MAXIMA_LISP/@CLISP_RUNTIME@" $MAXIMA_LISP_OPTIONS $CLISP_OPTIONS -q -M "$maxima_image_base.mem" "" -- "$arg1" "$arg2" "$arg3" "$arg4" "$arg5" "$arg6" "$arg7" "$arg8" "$arg9"
       fi
     else
       if [ -x "$maxima_image_base" ]; then
