--- maxima-5.10.0/interfaces/xmaxima/Tkmaxima/Menu.tcl.xdg_open	2006-09-11 10:38:01.000000000 -0500
+++ maxima-5.10.0/interfaces/xmaxima/Tkmaxima/Menu.tcl	2006-10-02 11:46:43.000000000 -0500
@@ -215,9 +215,9 @@
 	}
     } else {
 	
-	set selectedbrowser mozilla
+	set selectedbrowser xdg-open
 
-	foreach b { mozilla konqueror epiphany firefox netscape } {
+	foreach b { xdg-open htmlview firefox konqueror mozilla epiphany netscape } {
 	    if { ! [catch {exec which $b} ] } {
 		set selectedbrowser $b
 		break } }
--- maxima-5.10.0/interfaces/xmaxima/Tkmaxima/Constants.tcl.xdg_open	2006-08-03 08:21:57.000000000 -0500
+++ maxima-5.10.0/interfaces/xmaxima/Tkmaxima/Constants.tcl	2006-10-02 11:48:30.000000000 -0500
@@ -143,9 +143,9 @@
     image/gif  netmath
     image/png  netmath
     image/jpeg netmath
-    application/postscript "ghostview -safer %s"
-    application/pdf "acroread %s"
-    application/x-dvi "xdvi %s"
+    application/postscript "xdg-open %s"
+    application/pdf "xdg-open %s"
+    application/x-dvi "xdg-open %s"
 }
 set maxima_priv(imagecounter) 0
 
--- maxima-5.10.0/src/plot.lisp.xdg_open	2006-08-26 06:13:40.000000000 -0500
+++ maxima-5.10.0/src/plot.lisp	2006-10-02 11:46:43.000000000 -0500
@@ -75,7 +75,7 @@
                         ((mlist) $plot_realpart nil)
                         ))
 
-(defvar $viewps_command  "(ghostview \"~a\")")
+(defvar $viewps_command  "(xdg-open \"~a\")")
 
 ;; $plot_realpart option is false by default but *plot-realpart* is true because coerce-float-fun
 ;; is used outside of plot package too.
