--- maxima/interfaces/emacs/emaxima/maxima.el	2004-05-20 18:12:56.000000000 +0000
+++ maxima-new/interfaces/emacs/emaxima/maxima.el	2004-08-06 01:56:50.000000000 +0000
@@ -1628,6 +1628,9 @@
           (beginning-of-line)
 	  (while (re-search-forward 
                   (concat "(" maxima-inchar "[0-9]+).*\r") pmark t)
+	    (replace-match "" t t))
+          (beginning-of-line)
+	  (while (re-search-forward "\r" pmark t)
 	    (replace-match "" t t))))))
 
 (defun inferior-maxima-wait-for-output ()
