--- maxima-5.10.99rc2/interfaces/xmaxima/Tkmaxima/Menu.tcl.xdg_utils	2006-12-09 13:52:11.000000000 -0600
+++ maxima-5.10.99rc2/interfaces/xmaxima/Tkmaxima/Menu.tcl	2006-12-13 07:15:16.000000000 -0600
@@ -245,9 +245,9 @@
 	}
     } else {
 	
-	set selectedbrowser mozilla
+	set selectedbrowser xdg-open 
 
-	foreach b { firefox mozilla konqueror epiphany galeon amaya netscape } {
+	foreach b { xdg-open htmlview firefox mozilla konqueror epiphany galeon amaya netscape } {
 	    if { ! [catch {exec which $b} ] } {
 		set selectedbrowser $b
 		break } }
