diff -up maxima-5.29.0/src/displa.lisp.wxmaxima_compat maxima-5.29.0/src/displa.lisp
--- maxima-5.29.0/src/displa.lisp.wxmaxima_compat	2012-12-03 01:20:18.000000000 -0600
+++ maxima-5.29.0/src/displa.lisp	2012-12-08 18:12:40.500994027 -0600
@@ -1128,6 +1128,7 @@
       (dimension-nary form result)))
 
 (displa-def mlabel dim-mlabel 0 0)
+(setf (get 'mlabel 'wxxml) 'wxxml-mlable) ;; backwards-compatibility for wxMaxima
 
 (defvar *display-labels-p* t)
 
