--- maxima-5.9.1.9rc4/src/Makefile.am.sbcl	2005-05-20 01:14:01.000000000 -0500
+++ maxima-5.9.1.9rc4/src/Makefile.am	2005-10-06 11:31:04.118223968 -0500
@@ -115,7 +115,7 @@
 
 
 if SBCL
-EXECUTESBCL = $(SBCL_NAME) --noinform --noprint --eval
+EXECUTESBCL = $(SBCL_NAME) --noinform --noprint --disable-debugger --eval
 all-local: binary-sbcl/maxima.core
 install-exec-local: install-sbcl
 uninstall: uninstall-sbcl
--- maxima-5.9.1.9rc4/src/Makefile.in.sbcl	2005-10-02 12:55:30.000000000 -0500
+++ maxima-5.9.1.9rc4/src/Makefile.in	2005-10-06 11:31:23.267312864 -0500
@@ -169,7 +169,7 @@
 
 @ACL6_TRUE@EXECUTEACL6 = $(ACL6_NAME) -batch
 
-@SBCL_TRUE@EXECUTESBCL = $(SBCL_NAME) --noinform --noprint --eval
+@SBCL_TRUE@EXECUTESBCL = $(SBCL_NAME) --noinform --noprint --disable-debugger --eval
 
 @GCL_TRUE@EXECUTEGCL = $(GCL_NAME) -batch -eval
 
