--- maxima-5.9.3.99rc1/src/plot.lisp.evince	2006-07-16 15:42:04.000000000 -0500
+++ maxima-5.9.3.99rc1/src/plot.lisp	2006-08-01 09:06:18.000000000 -0500
@@ -75,7 +75,7 @@
                         ((mlist) $plot_realpart nil)
                         ))
 
-(defvar $viewps_command  "(ghostview \"~a\")")
+(defvar $viewps_command  "(evince \"~a\")")
 
 ;; $plot_realpart option is false by default but *plot-realpart* is true because coerce-float-fun
 ;; is used outside of plot package too.
--- maxima-5.9.3.99rc1/interfaces/xmaxima/Tkmaxima/Constants.tcl.evince	2006-07-29 04:27:34.000000000 -0500
+++ maxima-5.9.3.99rc1/interfaces/xmaxima/Tkmaxima/Constants.tcl	2006-08-01 09:07:23.000000000 -0500
@@ -143,8 +143,8 @@
     image/gif  netmath
     image/png  netmath
     image/jpeg netmath
-    application/postscript "ghostview -safer %s"
-    application/pdf "acroread %s"
+    application/postscript "evince %s"
+    application/pdf "evince %s"
     application/x-dvi "xdvi %s"
 }
 set maxima_priv(imagecounter) 0
