--- maxima-5.11.99rc2/src/Makefile.in.purify	2007-04-18 23:32:54.000000000 -0500
+++ maxima-5.11.99rc2/src/Makefile.in	2007-04-29 21:54:44.000000000 -0500
@@ -2031,7 +2031,7 @@
 @SBCL_TRUE@	test -d binary-sbcl/numerical || mkdir binary-sbcl/numerical
 @SBCL_TRUE@	test -d binary-sbcl/numerical/slatec || mkdir binary-sbcl/numerical/slatec
 @SBCL_TRUE@	$(EXECUTESBCL) '(progn $(LOADDEFSYSTEM) $(DEFSYSTEMCOMPILE) (sb-ext:quit))' && \
-@SBCL_TRUE@	$(EXECUTESBCL) '(progn $(LOADDEFSYSTEM) $(DEFSYSTEMLOAD) (sb-ext:save-lisp-and-die "binary-sbcl/maxima.core" :purify t) (sb-ext:quit))'
+@SBCL_TRUE@	$(EXECUTESBCL) '(progn $(LOADDEFSYSTEM) $(DEFSYSTEMLOAD) (sb-ext:save-lisp-and-die "binary-sbcl/maxima.core" :purify) (sb-ext:quit))'
 
 @SBCL_TRUE@install-sbcl:
 @SBCL_TRUE@	$(mkinstalldirs) $(DESTDIR)$(verpkglibdir)/binary-sbcl
